### PR TITLE
qa_crowbarsetup: Configure ceph testsuite for SES4

### PR DESCRIPTION
**Pull request by toabctl**
The testsuite from the storage3 branch needs to be configured
differently. So split out the configuration into different functions.

### CODE REVIEW

**Comment by toabctl** [comment]
Note: This is still expected to fail with a sigle failure in the ```test_ceph_version``` test.

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3920,6 +3920,57 @@ EOF
     safely zypper -n in ca-certificates-suse
 }
 
+function ceph_testsuite_configure_storage3
+{
+    # this is the configuration for the tests from the storage2 branch
```
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: storage3?

Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: fixed

**Comment by toabctl** [comment]
Hm. doesn't work...

**Comment by toabctl** [comment]
rebased so https://github.com/SUSE-Cloud/automation/pull/1380 is included now.

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -3922,6 +3922,61 @@ EOF
     safely zypper -n in ca-certificates-suse
 }
 
+function ceph_testsuite_configure_storage3
+{
+    # this is the configuration for the tests from the storage3 branch
+    yaml_allnodes=`echo $cephmons $cephosds | sed "s/ /\n/g" | sed "s/\..*//g" | sort -ru`
+    yaml_osds=`echo $cephosds | sed "s/ /\n/g" | sed "s/\..*//g" | sort -ru`
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: this does not work in case there are multiple osds per node. see the code in line 3974 how to do it properly

**Pull request merged**